siliconcompiler/schema/schema_cfg.py

```diff
@@ -1270,7 +1270,8 @@ def schema_datasheet(cfg, name='default', mode='default'):
             example=[
                 "cli: -datasheet_package_drawing 'abcd p484.pdf'",
                 "api: chip.set('datasheet', 'package', 'abcd', 'drawing', 'p484.pdf')"],
-            schelp="""Datasheet: package drawing.""")
+            schelp="""Mechanical package outline for documentation purposes.
+            Common file formats include PDF, DOC, SVG, PNG.""")
```

Comment:
`Common file formats include PDF, DOC, SVG, PNG` -> `Common file formats include PDF, DOC, SVG, and PNG`